### PR TITLE
silence coursier log when supershell is off

### PR DESCRIPTION
**Pull request by eed3si9n**
CI logs are filled up with Downloaded, so I am turning the Coursier log off.

Ref https://github.com/coursier/coursier/pull/872

### CODE REVIEW

File: main/src/main/scala/sbt/internal/librarymanagement/CoursierRepositoriesTasks.scala [hunk 1]
```diff
@@ -120,6 +121,36 @@ private[sbt] object CoursierRepositoriesTasks {
       }
     }
 
+  private val pluginIvySnapshotsBase = Resolver.SbtRepositoryRoot.stripSuffix("/") + "/ivy-snapshots"
+
+  def coursierSbtResolversTask: Def.Initialize[sbt.Task[Seq[Resolver]]] = Def.task {
```
Comment (eatkins) [inline main/src/main/scala/sbt/internal/librarymanagement/CoursierRepositoriesTasks.scala:126]: Doesn't really matter but the 'coursier' in `coursierSbtResolversTask` seems redundant in the context of CoursierRepositoriesTask.

File: main/src/main/scala/sbt/Defaults.scala [hunk 2]
```diff
@@ -243,7 +243,7 @@ object Defaults extends BuildCommon {
       // coursier settings
       csrExtraCredentials :== Nil,
       csrLogger := LMCoursier.coursierLoggerTask.value,
-      csrCachePath :== LMCoursier.defaultCacheLocation,
```
Comment (eatkins) [inline main/src/main/scala/sbt/Defaults.scala:246]: I like s/Path/Directory here.

File: main/src/main/scala/sbt/internal/LMCoursier.scala [hunk 3]
```diff
@@ -15,9 +15,14 @@ import sbt.librarymanagement._
 import Keys._
 import sbt.internal.librarymanagement.{ CoursierArtifactsTasks, CoursierInputsTasks }
 import sbt.util.Logger
+import sbt.io.syntax._
 
 private[sbt] object LMCoursier {
-  def defaultCacheLocation: File = CoursierDependencyResolution.defaultCacheLocation
+  def defaultCacheLocation: File =
+    sys.props.get("sbt.coursier.home") match {
```
Comment (eatkins) [inline main/src/main/scala/sbt/internal/LMCoursier.scala:22]: What is the expected behavior if the user passes in a symlink?

**Review by eatkins — APPROVED** [review]
LGTM overall.

File: main/src/main/scala/sbt/internal/librarymanagement/CoursierRepositoriesTasks.scala (hunk 1 above)
Comment (eed3si9n) [reply main/src/main/scala/sbt/internal/librarymanagement/CoursierRepositoriesTasks.scala:126]: In the original sbt-coursier code, `coursierResolversTask` implements `coursierResolvers` key, and this code tries to be somewhat consistent with that style so the maintenance is easier.

File: main/src/main/scala/sbt/internal/LMCoursier.scala (hunk 3 above)
Comment (eed3si9n) [reply main/src/main/scala/sbt/internal/LMCoursier.scala:22]: I'm guessing that it should use the full path of symlink without following?

**Pull request merged**